ingest/src/main/scala/geotrellis/osme/ingest/HttpSlippyTileReader.scala

```diff
@@ -0,0 +1,28 @@
+package geotrellis.osme.ingest
+
+import geotrellis.vector._
+import geotrellis.raster._
+import geotrellis.raster.io.geotiff._
+import geotrellis.spark._
+import geotrellis.spark.io.s3._
+import geotrellis.spark.io.slippy._
+import geotrellis.util.Filesystem
+
+import org.apache.commons.io.FileUtils
+import org.apache.commons.io.filefilter._
+import org.apache.commons.io.IOUtils._
+import org.apache.spark._
+import org.apache.spark.rdd._
+import java.net._
+import java.io.File
+
+class HttpSlippyTileReader[T](pathTemplate: String)(fromBytes: (SpatialKey, Array[Byte]) => T) extends SlippyTileReader[T] {
```

Comment:
This should be moved over to the `core` project to be in package `geotrellis.osme.core`